custom_md_loader/pages/CMTMigrator.page

```diff
@@ -0,0 +1,256 @@
+<!--
+ * Copyright (c) 2016, salesforce.com, inc.
+ * All rights reserved.
+ * Licensed under the BSD 3-Clause license. 
+ * For full license text, see LICENSE.txt file in the repo root  or https://opensource.org/licenses/BSD-3-Clause
+ */
+-->
+<apex:page controller="MetadataMigrationController" docType="html-5.0" sidebar="false"  action="{!onLoad}">
+
+	<body class="homeTab">
+
+      <apex:form id="form">
+    	
+       <apex:pageBlock title="Messages" rendered="{!isMessage == true}">
+            <apex:pageMessages />
+        </apex:pageBlock>
+
+		<apex:pageBlock title="Custom Settings/Custom Objects to Custom Metadata Types Migration">
```

Comment:
Section needs some rewriting to be more non-coding-user-friendly.

Reply:
+ mention of support for Custom objects and Custom list settings, but not custom hierarchy settings.